src/index.js

```diff
@@ -0,0 +1,155 @@
+import _ from 'lodash';
+import './index.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+
+  return element;
+}
+
+document.body.appendChild(component());
```

Comment:
- [ ] Kindly consider removing this function as it's not relevant to the project requirement for it make you UI not match the provided wireframe.
![image](https://github.com/Omwomo/to-do-list-project/assets/97100091/ff889af9-748e-4f2d-b642-1528c0c953d8)